phpBB/styles/prosilver/template/posting_buttons.html

```diff
@@ -105,6 +105,16 @@
 	<button type="button" class="button button-icon-only bbcode-color" name="bbpalette" id="bbpalette" value="{L_FONT_COLOR}" onclick="change_palette();" title="{L_BBCODE_S_HELP}">
 		<i class="icon fa-tint fa-fw" aria-hidden="true"></i>
 	</button>
+	<!-- IF S_SMILIES_ALLOWED -->
+	<button type="button" class="button button-icon-only smiley_button" accesskey="y" name="addsmiley" title="{L_SMILEYS}">
```

Comment:
Finally I found the reason for that unalignment of that pointer. It wasn't because of using correct dropdown.
Check this.
![image](https://cloud.githubusercontent.com/assets/12277453/25341373/40b0d736-2926-11e7-8806-4f5f29c01fa2.png)
![image](https://cloud.githubusercontent.com/assets/12277453/25341467/8115cbce-2926-11e7-914e-0ba759048a98.png)
It's because of a style of 
fieldset.fields1 div {
    margin-bottom: 3px;
}
Dropdown pointer changes with this.